docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -417,7 +417,7 @@ At the time of this writing, Visual Studio supports **Docker Compose** and **Kub
 
 After you add orchestrator support to your solution in Visual Studio, you will also see a new node (in the `docker-compose.dcproj` project file) in Solution Explorer that contains the added docker-compose.yml files, as shown in Figure 5-8.
 
-![Screenshot of docker-compose node in Solution Explorer.](./media/docker-app-development-workflow/docker-compose-tree-node.png)
+![Screenshot of docker-compose node in Solution Explorer.](./media/docker-app-development-workflow/docker-compose-tree-node.PNG)
```

Comment:
This extension should remain lowercase and the file should be renamed (please).

Reply:
@gewarren Sure.

Reply:
@Youssef1313 I believe to rename casing on a file you need to delete it, commit that, then add it again in a new commit.

Reply:
@adegeo Yeah I did that in https://github.com/dotnet/docs/pull/23395/commits/8f4037db618fba5d96e2318f4836bf9d0c8ecd8d and https://github.com/dotnet/docs/pull/23395/commits/5cb5f6c756740421f4fd413888673cc18b33752c and other commits for other files too.

Reply:
For some reason the comment still popped up, so I thought it hadn't been done :grin: